Assets/MixedRealityToolkit-SDK/Inspectors/UX/Interactable/ThemeInspector.cs

```diff
@@ -930,25 +933,33 @@ public static void RenderThemeSettings(SerializedProperty themeSettings, Seriali
 
                 if (animatorCount < sProps.arraySize)
                 {
-                    InspectorUIUtility.DrawDivider();
-
                     // show theme properties
                     SerializedProperty easing = settingsItem.FindPropertyRelative("Easing");
                     SerializedProperty enabled = easing.FindPropertyRelative("Enabled");
 
-                    enabled.boolValue = EditorGUILayout.Toggle(new GUIContent("Easing", "should the theme animate state values"), enabled.boolValue);
-                    if (enabled.boolValue)
+                    SerializedProperty noEasing = settingsItem.FindPropertyRelative("NoEasing");
+                    if (!noEasing.boolValue)
                     {
-                        EditorGUI.indentLevel = indentOnSectionStart + 1;
-                        SerializedProperty time = easing.FindPropertyRelative("LerpTime");
-                        //time.floatValue = 0.5f;
-                        SerializedProperty curve = easing.FindPropertyRelative("Curve");
-                        //curve.animationCurveValue = AnimationCurve.Linear(0, 1, 1, 1);
+                        InspectorUIUtility.DrawDivider();
+                        enabled.boolValue = EditorGUILayout.Toggle(new GUIContent("Easing", "should the theme animate state values"), enabled.boolValue);
 
-                        time.floatValue = EditorGUILayout.FloatField(new GUIContent("Duration", "animation duration"), time.floatValue);
-                        EditorGUILayout.PropertyField(curve, new GUIContent("Animation Curve"));
+                        if (enabled.boolValue)
+                        {
+                            EditorGUI.indentLevel = indentOnSectionStart + 1;
+                            SerializedProperty time = easing.FindPropertyRelative("LerpTime");
+                            //time.floatValue = 0.5f;
```

Comment:
Can this (and line 952) be removed or uncommented?

Reply:
Removed unused commented code.